objects/src/transaction/executed_tx.rs

```diff
@@ -168,3 +174,22 @@ pub struct TransactionMeasurements {
     pub tx_script_processing: usize,
     pub epilogue: usize,
 }
+
+impl TransactionMeasurements {
+    /// Returns the total number of cycles spent executing the transaction.
+    pub fn total_cycles(&self) -> usize {
+        self.prologue
+            + self.notes_processing
+            + self.note_execution.iter().map(|(_, c)| c).sum::<usize>()
```

Comment:
Actually, I just realized not `note_processing` is already a sum of all individual notes. So, we can delete line 183.